vj4/template.py

```diff
@@ -59,27 +59,18 @@ def __init__(self):
                          hoedown.HTML_HARD_WRAP)  # Render each linebreak as <br>.
 
 
-MARKDOWN_PRERENDER = [
-  (r'\(vijos\:\/\/fs\/([0-9a-f]{40,})\)',
-   lambda m: '(' + options.cdn_prefix.rstrip('/') + '/fs/' + m.group(1) + ')')
-   # TODO(twd2): reverse_url
-]
-MARKDOWN_PRERENDER_COMPILED = []
+FS_RE = re.compile(r'\(vijos\:\/\/fs\/([0-9a-f]{40,})\)')
 
-for regex, repl in MARKDOWN_PRERENDER:
```

Comment:
This is a good example - don't write "live code" (code that interpret by itself)